Documentation/Introduction/Index.rst

```diff
@@ -95,38 +103,38 @@ templates and are thus fully customizable.
 .. _screenshots:
 
 Screenshots
------------
+===========
 
-Some examples of Kitodo.Presentation in action on the `digital collection of SLUB Dresden <http://digital.slub-dresden.de/>`_:
+Some examples of Kitodo.Presentation in action on the `Digital Collection of SLUB Dresden <https://digital.slub-dresden.de/>`_:
 
-.. figure:: ../Images/Introduction/KitodoPresentaion-SLUB1.PNG
-   :width: 500px
+http://digital.slub-dresden.de/id457052678
+
+.. figure:: ../Images/Introduction/Digitale\ Sammlungen\ \ Colloquia\ Lutheri\ conscripta\ a\ quibusdam\ et\ alia\ quaedam\ addita\ sunt.\ Thesaurus\ theologiae\ -\ Mscr.Dresd.A.180.d.png
```

Comment:
The image links are broken. Should they be kept here?

Reply:
Maybe the links look broken, but they a properly escaped.

That's what the [renderer ](https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/RenderingDocs/Quickstart.html#render-documenation-with-docker)makes out of it:

![DlfDocumentationImage](https://user-images.githubusercontent.com/3354500/112840544-c12bb900-909f-11eb-960d-8a9fceae11a7.png)

Reply:
Ah ok, because in file view was there no displayed image.